content/news/new-project-logo.md

```diff
@@ -0,0 +1,43 @@
+Today we're pleased to show off a project that we have been working on for some time: a new logo for OpenRA!
+
+<div style="text-align:center" markdown="1">
+![](/images/news/20161223-new-logo.png)
+</div>
+
+This logo will be used to represent the OpenRA project as a whole, with the Soviet star logo being kept for the Red Alert mod.
+If community feedback is positive then we plan to roll this out with the next OpenRA release.
+
+Love it? Hate it? Let us know what you think in the comments below!
+
+### How does it look?
+
+We have developed mockups of how the logo will look in several important contexts:
+
+<div style="text-align:center; margin: 0 -60px" markdown="1">
+![](/images/news/20161223-new-logo-modchooser.png)<br />
+A preview of the ingame mod chooser with the new project logo.
+</div>
+
+<div style="text-align:center" markdown="1">
+![](/images/news/20161223-new-logo-launchers.png)<br />
+OpenRA icons in your operating system of choice.
+</div>
+
+<div style="text-align:center" markdown="1">
+![](/images/news/20161223-new-logo-facebook.png)<br />
+The new project logo on Facebook.
+</div>
+
+### Why do we need a new logo?
+
+Using the same logo for OpenRA (the project) and the Red Alert mod causes a lot of confusion, both for new players who may not know that OpenRA contains multiple separate games and for community-run events that need separate logos to distinguish between OpenRA (the project) and the three main mods.  These problems would only get worse in the future when the Tiberian Sun mod is eventually released.  Another issue is that the Soviet star symbol has strong symbolism associated with it, which makes it difficult for us to use it in contexts where people may not be familiar with OpenRA or Red Alert in general.
+
+### But why this design?
+
+We started the design process with three key requirements:
+
+* Reflect OpenRA's identity as a modern RTS game that leverages classic gameplay and artwork.
+* Not tied to a specific game or universe (so no mammoth tanks, tiberium, or faction logos).
+* Be capable of representing in a SVG vector format for the website and other media.
```

Comment:
This sentence sounds weird to my non-native ears. Isn't the "representing" missing an object?